blackbox/docs/sql/reference/alter_user.txt

```diff
@@ -0,0 +1,45 @@
+.. highlight:: psql
+.. _ref-alter-user:
+
+==============
+``ALTER USER``
+==============
+
+Alter an existing database user.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+::
+
+    ALTER USER
+      { SET ( user_parameter = value [, ...] }
```

Comment:
missing closing parenthesis. also I think as as long as there is only one option, surrounding brackets should be removed.